lib/components/SCard.vue

```diff
@@ -44,6 +45,7 @@ export default defineComponent({
 
   setup(props) {
     const isCollapsed = ref(false)
+    const showFooter = computed(() => get(props.footer?.actions ?? []).length > 0)
```

Comment:
I think this should be

```suggestion
    const showFooter = computed(() => (get(props.footer?.actions) ?? []).length > 0)
```

So I guess below is a little more readable.

```suggestion
    const showFooter = computed(() => {
      const actions = get(props.footer?.actions) ?? []
      
      return actions.length > 0
    })
```